middleware/index.ts

```diff
@@ -4,7 +4,9 @@ import { ConnectionConfig,
         ForceApi, 
         Logger, 
         UnitOfWork, 
-        FunctionInvocationRequest, 
+        SObject, 
+        SuccessResult, 
+        ErrorResult,
         UserContext as SdkUseContext,
```

Comment:
`SdkUserContext`?  Missing "r".

Reply:
@cwallsfdc , we set it on the context object at

https://github.com/forcedotcom/sf-fx-middleware/pull/9/files#diff-5f090c7ace3f484d222605005bc0ebbcR145

where it was created above (around line 134)